akka-actor-testkit-typed/src/main/scala/akka/actor/testkit/typed/internal/TestProbeImpl.scala

```diff
@@ -141,13 +140,21 @@ private[akka] final class TestProbeImpl[M](name: String, system: ActorSystem[_])
     }
   }
 
+  override def receiveOne(): M = receiveOne(remainingOrDefault)
+
+  override def receiveOne(max: java.time.Duration): M = receiveOne(max.asScala)
+
+  def receiveOne(max: FiniteDuration): M =
+    receiveOne_internal(max).
```

Comment:
`.dilated` missing
```
receiveOne_internal(max.dilated)
```

Dilation is a factor that can be used to increase test timeouts, for example in slow CI environments. Default is 1.0